src/clusterfuzz/_internal/issue_management/issue_filer.py

```diff
@@ -456,6 +456,9 @@ def file_issue(testcase,
   for cc in ccs:
     issue.ccs.add(cc)
 
+  # Apply extension fields.
+  issue.apply_extension_fields(policy.extension_fields)
```

Comment:
Can you add `apply_extension_fields` to the base `Issue` class with a default no-op impl?

Reply:
> Can you add `apply_extension_fields` to the base `Issue` class with a default no-op impl?

This was already added here- https://github.com/google/clusterfuzz/pull/3442/files#diff-3ef2899e989a84cf609a31a2e54dcfbbe582b71efb516af9a8f8dbecd4a6e703
(unless you meant another place)

Reply:
Ah, sorry I missed that.